lib/fortnox/api.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 require 'set'
-require 'dry-struct'
```

Comment:
Not sure why we required `dry-struct` here before. We are not using that in this file. What we are using is `dry-container` and `dry-configurable`...

Reply:
Probably a leftover from refactoring ...